tests/integration_tests/clouds.py

```diff
@@ -143,12 +143,30 @@ def _get_cloud_instance(self):
         )
 
 
+def _mount_source(instance):
+    container_path = '/usr/lib/python3/dist-packages/cloudinit'
+    format_variables = {
+        'name': instance.name,
+        'cloudinit_path': cloudinit.__path__[0],
```

Comment:
While I'm nitpicking, both paths here are "cloud-init path"s in a sense: perhaps `source_path`?